incubator/haproxy-ingress/templates/controller-daemonset.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
```

Comment:
Please refer [names-and-labels](https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md#names-and-labels) to set labels.

Reply:
Done